CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 ## [v0.27.0-rc0](https://github.com/thanos-io/thanos/tree/release-0.27) - Release in progress
 
 ### Fixed
+- [#5453](https://github.com/thanos-io/thanos/pull/5453) Compact: Fix crash during 1h downsampling of 5m resolution blocks
```

Comment:
* This should go to the "Unrelased" section
* Let's be obvious what we are actually adding, so:

```suggestion
- [#5453](https://github.com/thanos-io/thanos/pull/5453) Compact: Skip erroneous empty non `*AggrChunk` chunks during 1h downsampling of 5m resolution blocks.
```